### ISSUE TITLE
events.js:180 Unhandled error event

### ISSUE THREAD

**Issue by BillyNate** [amark/gun#787]
I did a clone on `master`, installed the required node modules and `npm start` to try out GUN

Unfortunately I'm getting the error mentioned in the title of this issue.

This is what's shown in the console:
```
AXE enabled.
Relay peer started on port 8765 with /gun
events.js:180
      throw er; // Unhandled 'error' event
      ^

Error: write ENOTSUP
    at ChildProcess.target._send (internal/child_process.js:806:20)
    at ChildProcess.<anonymous> (internal/child_process.js:623:16)
    at ChildProcess.emit (events.js:208:15)
    at emit (internal/child_process.js:876:12)
    at processTicksAndRejections (internal/process/task_queues.js:77:11)
Emitted 'error' event at:
    at ChildProcess.<anonymous> (internal/cluster/worker.js:27:12)
    at ChildProcess.emit (events.js:203:13)
    at internal/child_process.js:810:39
    at processTicksAndRejections (internal/process/task_queues.js:75:11) {
  errno: 'ENOTSUP',
  code: 'ENOTSUP',
  syscall: 'write'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gun@0.2019.711 start: `node examples/http.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gun@0.2019.726 start script.
```

By doing checkouts on previous commits I was able to figure out this error only pops up in commits from ab36283b2e8943d31f087303f3eb519f8d5553b1 and later.  
This is on Windows 10 x64 with Node 12.8.0 and NPM 6.10.3

If there's any more information I can provide, please let me know.

**Comment by LiamKarlMitchell** [comment]
Still happening
Node V12.2.0
Windows 10 x64

**Comment by BillyNate** [comment]
I dived into the code that changed between 0c718c8 and ab36283.  
The line [`if(true !== opt.multicast){ return } // disable multicast by default for now.`](https://github.com/amark/gun/blob/0c718c8658634f11232b779a61788e6d0a8cf22a/lib/multicast.js#L7) disabled everything below this line before ab36283.  
As it turns out, the error was "only" triggered on [`socket.bind(udp.port)`](https://github.com/amark/gun/blob/0c718c8658634f11232b779a61788e6d0a8cf22a/lib/multicast.js#L18).
Some Googling revealed a [StackOverflow question about UDP multicast failing on Windows 10](https://stackoverflow.com/questions/44141668/udp-multicast-failing-nodejs-windows-10), including a fix 🎉 

When I changed `socket.bind(udp.port)` to
```
socket.bind({port: udp.port, exclusive: true}, function(){
  socket.setBroadcast(true);
  socket.setMulticastTTL(128);
  socket.addMembership(udp.address);
});
```
the error went away! 😄 

I could turn this into a PR in you want, but it's only tested on W10, and I haven't tested if multicast actually works, nor if any other bugs have been introduced by this "fix".
